easybuild/easyconfigs/e/ecCodes/ecCodes-2.7.0-gmvolf-17.02-Python-2.7.13.eb

```diff
@@ -0,0 +1,38 @@
+# File modified by MaKra - August 2017 (CSCS)
+easyblock = 'CMakeMake'
+
+name = 'ecCodes'
+version = '2.7.0'
+versionsuffix='-Python-2.7.13'
+
+homepage = 'https://software.ecmwf.int/wiki/display/GRIB/Home'
+description = """ecCodes is a package developed by ECMWF which provides an
+application programming interface and a set of tools for decoding and encoding
+messages in different formats"""
+
+toolchain = {'name': 'gmvolf', 'version': '17.02'}
+
+source_urls = ['https://software.ecmwf.int/wiki/download/attachments/45757960/']
```

Comment:
```
Page Not Found
```

Reply:
yes, I couldn't find a solution for this problem yet. My user is able to download the code, but not jenkins